MaterialLib/MPL/VariableType.h

```diff
@@ -42,19 +42,20 @@ using Tensor = std::array<double, 9>;
 /// is missing, simply add it somewhere at the list, but above the last entry.
 enum class Variable : int
 {
-    concentration,
-    phase_pressure,
-    effective_pore_pressure_rate,
     capillary_pressure,
```

Comment:
I like the new consistent sorting in the enum. It would be nice and even more consistent if you would sort also the corresponding translation function `convertStringToVariable` in the cpp file. ;-)

Reply:
Kept it for a cleanup session. Especially there half of the enum is not handled...

Reply:
That is fine for me.